docs/asciidoc/front_matter.adoc

```diff
@@ -11,16 +20,102 @@ The primary functionality of DRS is to map a logical ID to a means for physicall
+
+===== For DRS Clients
+
+A DRS client parses the DRS URI compact identifier components to extract the prefix and the accession, and then uses meta-resolver APIs to locate the actual DRS server. For example, here are the client resolution steps if the URI is:
+
+    drs://drs.42:314159
+
+1) The client parses the string to extract the prefix of `drs.42` and the accession of `314159`, using the first occurrence of a colon (":") character after the initial `drs://` as a delimiter. (The colon character is not allowed in a Hostname-based DRS URI, making it easy to tell them apart.)
+
+2) The client makes API calls to a meta-resolver to look up the URL pattern for the namespace. (See <<_appendix_compact_identifier_based_uris>> for details.) The URL pattern is a string containing a `{$id}` parameter, such as:
+
+    https://drs.myexample.org/ga4gh/drs/v1/objects/{$id}
+
+3) The client generates a DRS URL from the URL template by replacing {$id} with the accession it extracted in step 1. It then makes a GET request to the DRS server:
+
+    GET https://drs.myexample.org/ga4gh/drs/v1/objects/314159
+
+4) The client follows any HTTP redirects returned in step 3, in case the resolver goes through an extra layer of redirection.
+
+For performance reasons, DRS clients SHOULD cache the URL pattern returned in step 2, with a suggested 24 hour cache life.
+
+==== Choosing a URI Style
+
+DRS servers can choose to issue either hostname-based or compact identifier-based DRS URIs, and can be confident that compliant DRS clients will support both.  DRS clients *must* be able to accommodate both URI types. Tradeoffs that DRS server builders, and third parties who need to cite DRS objects in datasets, workflows or elsewhere, may want to consider include:
+
+.Table Choosing a URI Style
+|===
+| |Hostname-based |Compact Identifier-based
+
+|URI Durability
+|URIs are valid for as long as the server operator maintains ownership of the published DNS address. (They can of course point that address at different physical serving infrastructure as often as they’d like.)
+|URIs are valid for as long as the server operator maintains ownership of the published compact identifier resolver namespace. (They also depend on the meta-resolvers like identifiers.org/n2t.net remaining operational, which is intended to be essentially forever.)
+
+|Client Efficiency
+|URIs require minimal client logic, and no network requests, to resolve.
+|URIs require small client logic, and 1-2 cacheable network requests, to resolve.  Citation practice is consistent with publisher instructions.
```

Comment:
Please delete  "Citation practice is consistent with publisher instructions."  As discussed, the only reference we've seen is a vague suggestion, and one that neither URI style conforms to.

Reply:
Done